### PR TITLE
Fixed '530353 - Generate Overrides fix fails'

### PR DESCRIPTION
**Pull request by mkrueger**
Workaround for missing UI.

### CODE REVIEW

File: main/src/addins/MonoDevelop.Refactoring/MonoDevelop.CodeIssues/AnalyzersFromAssembly.cs [hunk 1]
```diff
@@ -90,6 +90,10 @@ internal void AddAssembly (System.Reflection.Assembly asm, bool force = false)
 			try {
 				foreach (var type in asm.GetTypes ()) {
 
+					//HACK: Workaround for missing UI
+					if (type == typeof (Microsoft.CodeAnalysis.GenerateOverrides.GenerateOverridesCodeRefactoringProvider))
```
Comment (Therzok) [inline main/src/addins/MonoDevelop.Refactoring/MonoDevelop.CodeIssues/AnalyzersFromAssembly.cs]: This does not have the diagnosticanalyzer attribute?

**Comment by Therzok** [comment]
Can't we just import:
http://source.roslyn.io/#Microsoft.VisualStudio.LanguageServices/Implementation/PickMembers/VisualStudioPickMembersService.cs,14
http://source.roslyn.io/#Microsoft.CodeAnalysis.Features/GenerateOverrides/GenerateOverridesCodeRefactoringProvider.cs,16

?

File: main/src/addins/MonoDevelop.Refactoring/MonoDevelop.CodeIssues/AnalyzersFromAssembly.cs (hunk 1 above)
Comment (Therzok) [reply main/src/addins/MonoDevelop.Refactoring/MonoDevelop.CodeIssues/AnalyzersFromAssembly.cs]: Oh, it hits the `ExportCodeFixProviderAttribute` path.

**Comment by Therzok** [comment]
Yeah, I say we copy-paste http://source.roslyn.io/#Microsoft.CodeAnalysis.Features/GenerateOverrides/GenerateOverridesCodeRefactoringProvider.cs,16 until editorfeatures come in

and implement http://source.roslyn.io/#Microsoft.VisualStudio.LanguageServices/Implementation/PickMembers/VisualStudioPickMembersService.cs,14
to route via https://github.com/mono/monodevelop/blob/master/main/src/addins/CSharpBinding/MonoDevelop.CSharp.CodeGeneration/GenerateCodeWindow.cs

**Comment by mkrueger** [comment]
I made a task for that - proper fix takes weeks.

**Pull request merged**